Parsers/ASimDns/Parsers/ASimDnsMicrosoftNXlog.yaml

```diff
@@ -1,12 +1,12 @@
 Parser:
   Title: DNS activity ASIM parser for Microsoft DNS logs collected using NXlog
-  Version: '0.4'
-  LastUpdated: Dec 11 2022
+  Version: '0.5'
+  LastUpdated: Jul 10 2023
 Product:
   Name: MS DNS Events
 Normalization:
   Schema: Dns
-  Version: '0.1.3'
+  Version: '0.1.7'
 References:
 - Title: ASIM DNS Schema
   Link: https://aka.ms/ASimDnsDoc
```

Comment:
Can we add a new DNS document reference link here from where you have sourced these event info?

Reply:
Hi Varun (@vakohl),
I used the [page](https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-dns#schema-updates) that describes the updates between the different versions of the schema. The short link leads to the same page.

If you mean, where I got the list of DNS recource record IDs that I added starting from line 162, then the answer is the [Wikipedia page about DNS resource records](https://en.wikipedia.org/wiki/List_of_DNS_record_types).
Shall I add that to the list of references too?

Reply:
Do you any Microsoft Documentation where we can find same information? Wikipedia sometimes may not be accurate.

Reply:
I have not come across such a comprehensive list of DNS RR IDs in the MS related documentation. But I don't think that any of this information would be Microsoft specific.
The Wikipedia article lists a lot for [references](https://en.wikipedia.org/wiki/List_of_DNS_record_types#References), with the actual information being spread out in a number of RFC articles.

Reply:
can we add https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml instead of Wiki Link? this has the list you we are adding here.

Reply:
sure thing. fixed.